datacapture/src/main/java/com/google/android/fhir/datacapture/QuestionnaireReviewAdapter.kt

```
@@ -39,6 +39,7 @@ internal class QuestionnaireReviewAdapter :
             .inflate(R.layout.pagination_navigation_view, parent, false),
         )
       QuestionnaireEditAdapter.ViewType.Type.REPEATED_GROUP_HEADER -> TODO()
+      QuestionnaireEditAdapter.ViewType.Type.REPEATED_GROUP_ADD_BUTTON -> TODO()
```

wouldn't this just crash? and is it crashing now for the line above REPEATED_GROUP_HEADER?

unless we filter out these items beforehand?

if we do filter them out correctly beforehand, then here it should throw an error. (illegal state exception).